### ISSUE TITLE
np.int -> int

### ISSUE THREAD

**Issue by lorenzo-pino1** [LucaMalavolta/SLOPpy#10]
I am running SLOPpy in python 3.9 in a dedicated anaconda environment. My numpy version in this environment is 1.26.3.

Line 458 in the subroutine mcmc_fit_functions throws the following error:

```
*** AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

The easy fix is to change np.int with int as suggested in the error. I guess another option is to suggest a specific version of numpy to be installed (maybe it is alreadu, and I missed it.

**Comment by lorenzo-pino1** [comment]
Edit: there are several other occurrences throughout other routines (transmission_spectrum.py, transmission_spectrum_average.py, transmission_lightcurve_average.py, spectra_lightcurve.py, .... potentially others, I did not check this thoroughly)

**Comment by LucaMalavolta** [comment]
The issue has been fixed with version 1.3, thank you very much!
